container/container_unix.go

```diff
@@ -343,6 +343,17 @@ func (container *Container) BuildCreateEndpointOptions(n libnetwork.Network) ([]
 		createOptions = append(createOptions, libnetwork.EndpointOptionGeneric(genericOption))
 	}
 
+	/* FIXME: When restoring, the container should get the same IP address it had before, so
```

Comment:
Networks don't currently restore correctly at all, so this is only part of what needs to be done.